src/telemetry/graph.jsx

```diff
@@ -72,7 +72,6 @@ export default class TelemetryContainer extends React.Component {
             <a
               className="graph-title-link"
               ref={a => (this.graphTitleLink = a)}>
-              onCic
```

Comment:
Thanks, @klahnakoski  this was automatically added, I removed and rand yarn jest -u again. 

Then I ran yarn jest 
I didn't give any error. 

It should resolve the tests then.